lib/mayaUsd/ufe/ProxyShapeHierarchy.h

```diff
@@ -64,15 +64,23 @@ class MAYAUSD_CORE_PUBLIC ProxyShapeHierarchy : public Ufe::Hierarchy
 #endif
 
 #ifdef UFE_V2_FEATURES_AVAILABLE
+#if (UFE_PREVIEW_VERSION_NUM >= 3005)
+    Ufe::SceneItem::Ptr createGroup(const Ufe::PathComponent& name) const override;
+#else
     Ufe::SceneItem::Ptr
     createGroup(const Ufe::Selection& selection, const Ufe::PathComponent& name) const override;
+#endif
```

Comment:
adding guards against UFE v0.3.5 since selection is no longer passed to group command.